x-pack/plugins/secops/server/utils/logger.ts

```diff
@@ -4,20 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Server } from 'hapi';
-
 const LOGGING_TAGS = ['secops'];
 
+// Definition (except for the "() => any" at the end is from :
```

Comment:
Actually by introducing the `ban types` we can mirror this perfectly as the types have it. Seems compatible as well. I would update this comment.